pywbem_mock/_mockmofwbemconnection.py

```diff
@@ -28,30 +28,25 @@
 from __future__ import absolute_import, print_function
 
 from pywbem import CIMError, CIM_ERR_INVALID_PARAMETER, \
-    CIM_ERR_NOT_FOUND, CIM_ERR_ALREADY_EXISTS, CIM_ERR_FAILED, \
-    CIM_ERR_INVALID_SUPERCLASS, CIM_ERR_INVALID_NAMESPACE, CIMInstanceName
+    CIM_ERR_NOT_FOUND, CIM_ERR_FAILED, CIM_ERR_INVALID_SUPERCLASS
 from pywbem._nocasedict import NocaseDict
-from pywbem._mof_compiler import MOFWBEMConnection
+from pywbem._mof_compiler import BaseRepositoryConnection
 from pywbem._utils import _format
 from ._resolvermixin import ResolverMixin
 
 # Issue #2062 TODO/ks Remove this code and use the methods in _WBEMConnection
 # in their place
 
 
-class _MockMOFWBEMConnection(MOFWBEMConnection, ResolverMixin):
+class _MockMOFWBEMConnection(BaseRepositoryConnection, ResolverMixin):
     """
-    Create an adaption of the MOF compiler MOFWBEMConnection class so that to
-    directly use the attributes that represent the repository and implement a
-    modified CreateClass that resolves the new class. This directs the compiler
-    output directly  to the dictionaries for qualifiers, and instances in the
-    FakedWBEMConnection object and replaces the CreateClass with a local method
-    that allows resolving the created class before it is inserted into the
-    repository.
+    Create an adaption of the MOF compiler MOFWBEMConnection class to interface
```

Comment:
It is no longer an adaption of the MOFWBEMConnection class.

Reply:
DONE